Assets/Aircraft Physics/Example/Scripts/AirplaneController.cs

```diff
@@ -66,7 +70,12 @@ private void Update()
             //clamp
             flap = Mathf.Clamp(flap, 0f, Mathf.Deg2Rad * 40);
         }
-
+          if (Input.GetKeyDown(KeyCode.LeftAlt))
+        {
+            flap -= flapControlSensitivity;
+            //clamp
+            flap = Mathf.Clamp(flap, 0f, Mathf.Deg2Rad * 40);
+        }
```

Comment:
This is unnecessary, flaps only go downward, not upward in real airplanes. They are used for takeoff to get the plane off the ground because they add lift. They are used for landing to slow down the plane while keeping it airborne until its ready to land.

Also just press shift at the same time as hitting ctrl, this will make the flap go up to its original position.

Reply:
My bad, I didn't pay attention to what the shift key do, maybe add a in-game icon to say that the shift is enabled to help to visualize when the shift key is pressed.